nodejs/egg-pterodactyl-node-js-generic.json

```
@@ -4,23 +4,17 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2025-06-09T14:02:56+02:00",
+    "exported_at": "2025-10-20T09:57:28+08:00",
     "name": "node.js generic",
     "author": "parker@parkervcp.com",
     "description": "a generic node.js egg\r\n\r\nThis will clone a git repo. it defaults to master if no branch is specified.\r\n\r\nInstalls the node_modules on install. If you set user_upload then I assume you know what you are doing.",
     "features": null,
     "docker_images": {
-        "Nodejs 12": "ghcr.io\/pelican-eggs\/yolks:nodejs_12",
-        "Nodejs 14": "ghcr.io\/pelican-eggs\/yolks:nodejs_14",
-        "Nodejs 16": "ghcr.io\/pelican-eggs\/yolks:nodejs_16",
-        "Nodejs 17": "ghcr.io\/pelican-eggs\/yolks:nodejs_17",
-        "Nodejs 18": "ghcr.io\/pelican-eggs\/yolks:nodejs_18",
-        "Nodejs 19": "ghcr.io\/pelican-eggs\/yolks:nodejs_19",
-        "Nodejs 20": "ghcr.io\/pelican-eggs\/yolks:nodejs_20",
-        "Nodejs 21": "ghcr.io\/pelican-eggs\/yolks:nodejs_21",
-        "Nodejs 22": "ghcr.io\/pelican-eggs\/yolks:nodejs_22",
+        "Nodejs 24": "ghcr.io\/pelican-eggs\/yolks:nodejs_24",
         "Nodejs 23": "ghcr.io\/pelican-eggs\/yolks:nodejs_23",
-        "Nodejs 24": "ghcr.io\/pelican-eggs\/yolks:nodejs_24"
+        "Nodejs 22": "ghcr.io\/pelican-eggs\/yolks:nodejs_22",
+        "Nodejs 21": "ghcr.io\/pelican-eggs\/yolks:nodejs_21",
+        "Nodejs 20": "ghcr.io\/pelican-eggs\/yolks:nodejs_20"
```

```suggestion
        "Nodejs 20": "ghcr.io\/pelican-eggs\/yolks:nodejs_20",
        "Nodejs 21": "ghcr.io\/pelican-eggs\/yolks:nodejs_21",
        "Nodejs 22": "ghcr.io\/pelican-eggs\/yolks:nodejs_22",
        "Nodejs 23": "ghcr.io\/pelican-eggs\/yolks:nodejs_23",
        "Nodejs 23": "ghcr.io\/pelican-eggs\/yolks:nodejs_23",
        "Nodejs 24": "ghcr.io\/pelican-eggs\/yolks:nodejs_24"
```
Pterodactyl sorts them backwards.

---

ewwww. Sorry I reversed them coz It didn't make sense why it was flipped for one